### ISSUE TITLE
Generation of small packets in input buffer can cause timing issue

### ISSUE THREAD

**Issue by xross** [xmos/lib_xua#420]
An error in early feedback values can cause the input pipe buffer to be filled with lots of small, invalid, packets. This is not normally an issue since the host is not streaming. 

However, this can cause a timing issue since the overflow handling loops though the buffering removing packets. In some circumstances (i.e. adding some intensive processing via UserBufferManagement) this can lead to the AudioHub task being stalled for a time, the ports being stared of data and a bit-shift can occur on the I2S ports.

An error is troublesome if large, as possible with the undocumented, alpha feature FB_USE_REF_CLOCK. With the standard feedback code this is not an issue. (FB_USE_REF_CLOCK allows feedback to be generated based on the system clock, this is only a valid scheme if the audio clocks are related to the system clock)

Repeated unplugging of a self-powered device can trigger generation of this error.

To summarise, a timing issue can be generated under the following conditions : 

- unplug/plug events on self-powered device
- additional of intensive processing (via UserBufferManagement)
- use of FB_USE_REF_CLOCK (and appropriate clocking scheme)
